src/icons/file-icon/FileIcon.js.flow

```diff
@@ -189,7 +203,7 @@ type Props = {
 };
 
 const FileIcon = ({ dimension = 32, extension = '', title }: Props) => {
-    const IconComponent = Components[getFileIconComponent(extension)];
+    const IconComponent = Components[getFileIconComponent(extension.toLowerCase())];
```

Comment:
why are we lowercasing?

Reply:
In windows, file extensions are case insensitive, so it's possible for someone to upload a file with an uppercased extension. When I googled the heic type I found multiple examples of `.HEIC`, which I found out we currently don't handle without this change

Reply:
suggest doing the lower-casing in `getFileIconComponent` instead of at the call site